content/docs/capabilities/routing.mdx

```diff
@@ -407,6 +407,17 @@ routes:
       - http://myapp-srv-5:8080,10
 ```
 
+## Route matching order
+
+Pomerium processes routes in the order they are defined in its configuration. If you’re using Pomerium Core, then Pomerium will source its configuration from a configuration file.
```

Comment:
I wonder if it would also be good to include an example here where the order is important? e.g. a route with a path prefix and then another route for the same domain without any path prefix?